Package.swift

```diff
@@ -79,6 +83,8 @@ let SwiftFirebase =
                           "-Lthird_party/firebase-development/usr/libs/windows",
                           "-Lthird_party/firebase-development/usr/libs/windows/deps/app",
                           "-Lthird_party/firebase-development/usr/libs/windows/deps/app/external",
+                          "-Xlinker",
+                          "-ignore:4217"
```

Comment:
Was mostly getting annoyed seeing these in the build logs. I see we ignore them on other builds so I felt it was safe to do it here.